dotnet/docker/dotnet_release_app.dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim AS build_app
+
+ARG PERSONAL_ACCESS_TOKEN
+
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends  openssh-client
+
+RUN set -x \
+    && mkdir /app \
+    && git clone https://${PERSONAL_ACCESS_TOKEN}@github.com/DataDog/aas-dotnet-samples.git \
+    && cd aas-dotnet-samples/Junkyard/Junkyard.Web \
+    && dotnet publish -c Release --framework net8.0 -o /app
```

Comment:
The repo aas-dotnet-samples is private, so we need a way to clone it. Here I generated a PAT and use it. If you have another approaches (deploy keys) you would prefer, I'm all ears :)

Reply:
✔️